src/frontend/src/flows/addDevice/manage/addDeviceSuccess.test.ts

```diff
@@ -0,0 +1,61 @@
+import { renderAddDeviceSuccess } from "./addDeviceSuccess";
+import copyJson from "./addDeviceSuccess.json";
+
+describe("addDeviceSuccess", () => {
+  const deviceAlias = "Test device alias";
+
+  const { en } = copyJson;
+
+  const waitFor = (callback: () => void): Promise<void> =>
+    new Promise((resolve) => {
+      const start = Date.now();
+
+      const callCallback = () => {
+        if (Date.now() > start + 10 * 1000) {
+          resolve();
+          return;
+        }
+
+        try {
+          callback();
+          resolve();
+        } catch (err: unknown) {
+          // Ignore error and try again in a bit
+          setTimeout(callCallback, 100);
+        }
+      };
+
+      callCallback();
+    });
+
+  beforeAll(() => {
+    const pageContent = document.createElement("div");
+    pageContent.setAttribute("id", "pageContent");
+    document.body.appendChild(pageContent);
+  });
+
+  it("should render static content", async () => {
+    renderAddDeviceSuccess({
+      deviceAlias,
+    });
+
+    await waitFor(() => expect(document.body.textContent).toContain(en.title));
+    expect(document.body.textContent).toContain(en.explore);
+  });
```

Comment:
Wouldn't this be checked by screenshots anyway?

Reply:
Yes sure but when we develop locally we mostly run these unit test. That way we could have a quicker feedback on unexpected changes without waiting the CI to run.

At least in my workflow it's handy.

Reply:
I think you're testing the lit-html behavior here: either you've removed text on purpose and then the test will fail which you'll need to fix (annoying), or _something_ is failing to render the text, but really that's `lit-html` (which we don't want to test).

In which situation do you think this test would be beneficial?

Reply:
Removed